### DISCUSSION TITLE
Saving the last selected configuration for a project inside .vscode folder

### DISCUSSION THREAD

**Discussion by shaantalk** [onlyutkarsh/git-config-user-profiles#44] [Ideas]
Can we add a feature to store the last selected git profile for a project inside .vscode folder so that it remembers my choice and automatically sets that profile when I open that project

**Comment by onlyutkarsh** [comment]
hey @shaantalk - I just released a newer version (thanks to #45). The profile should automatically get loaded if username and email in your `.git\config` matches a profile in saved profiles. let me know if that solves it for you.
